File: Codesignal/Python/Feedback Review.py
from functools import reduce


def joinWord(a, b):
  try:
    if ((len(a) & len(b)) & (len(a)+len(b) <= 7)):
        return " ".join([a, b])
  except:
    return None


def feedbackReview(feedback, size):
    arr = feedback.split()
    return reduce(joinWord, arr)


#print(joinWord('as', 'pes'))

print(feedbackReview('This is an example feedback', 8))

File: Codesignal/Javascript/Feedback Review.js
function feedbackReview(feed, size) {
  return feed.split(' ').map((e,i, arr) => {
    let next = arr[i+1] ? arr[i+1] : ' '
    if(e.length + next.length <= size-1){
      arr.splice(i,1)
      return `${e} ${next}`
    } else return e
  })//.filter(String)
}



console.log(feedbackReview("This is an example feedback", 8))